### PR TITLE
Replace CSV in the host settlement to v2

### PR DESCRIPTION
**Pull request by znarf**
Part of opencollective/opencollective#4590

### CODE REVIEW

**Review by kewitz — APPROVED** [review]
LGTM. Tested locally and it works. :shipit:

File: cron/monthly/host-settlement.ts [hunk 1]
```diff
@@ -152,10 +144,11 @@ AND ts."status" != 'SETTLED'`,
       } (${host.currency})`,
     );
 
-    let csv;
-    if (transactions.length) {
-      csv = generateCSVFromTransactions(transactions);
-    }
+    const csv = await getCollectiveTransactionsCsv(host, {
```
[the remark this answers is not in the document]
Comment (znarf) [reply cron/monthly/host-settlement.ts:147]: I thought about it but didn't want to disrupt current process.

Does that sounds acceptable to you? Could be interesting to remove the dependency on S3.

[the remark this answers is not in the document]
Comment (kewitz) [reply cron/monthly/host-settlement.ts]: @znarf sure it is acceptable. The download and upload should be super fast.

[the remark this answers is not in the document]
Comment (znarf) [reply cron/monthly/host-settlement.ts]: My question was unclear, I was actually willing to explore your proposal.

After some extra input from @Betree, I moved forward with it.

**Pull request merged**